.config/nvim/lua/plugins/configs/diffview.lua

```diff
@@ -1,4 +1,41 @@
 return {
 	"sindrets/diffview.nvim",
 	cmd = { "DiffviewOpen" },
+	config = function()
+		local actions = require("diffview.actions")
+		require("diffview").setup({
+			keymaps = {
+				view = {
+					{ "n", "gf", false },
+					{ "n", "<C-w>gf", false },
+					{ "n", "<C-w><CR>", actions.goto_file_tab, { desc = "Open the file in a new tabpage" } },
```

Comment:
Add suggestion via octo.nvim
```suggestion
					{
                        "n",
                        "<C-w><CR>",
                        actions.goto_file_tab,
                        { desc = "Open the file in a new tabpage" }
                    },
```